index.js

```diff
@@ -1,6 +1,16 @@
 import { requireNativeComponent } from "react-native";
 
 export { default as VizbeeManager } from "./src/VizbeeManager";
+
+export {default as VizbeeUIConfiguration } from "./src/uiConfig/VizbeeUIConfiguration";
+export {default as VizbeeUICardConfiguration } from "./src/uiConfig/VizbeeUICardConfiguration";
+export {default as VizbeeCastAuthorizationCardConfiguration } from "./src/uiConfig/VizbeeCastAuthorizationCardConfiguration";
+export {default as VizbeeCastIntroductionCardConfiguration } from "./src/uiConfig/VizbeeCastIntroductionCardConfiguration";
+export {default as VizbeeSmartInstallCardConfiguration } from "./src/uiConfig/VizbeeSmartInstallCardConfiguration";
+export {default as VizbeeGuidedSmartInstallCardConfiguration } from "./src/uiConfig/VizbeeGuidedSmartInstallCardConfiguration";
+export {default as VizbeeMultiDeviceSmartInstallCardConfiguration } from "./src/uiConfig/VizbeeMultiDeviceSmartInstallCardConfiguration";
+export {default as VizbeeUICardType } from "./src/uiConfig/VizbeeUICardType";
```

Comment:
@shivareddy Above export should be corrected as follows - `export { VizbeeUICardType } from "./src/uiConfig/VizbeeUICardType";`

Reply:
Done